client/src/components/shared/NewInfo/Eurofunds.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import {compose, withState, withHandlers} from 'recompose'
+import ChevronUp from 'react-icons/lib/fa/chevron-up'
+import ChevronDown from 'react-icons/lib/fa/chevron-down'
+import {Badge, Button} from 'reactstrap'
+
+import ExternalLink from '../ExternalLink'
+import {ShowNumberCurrency} from '../../../services/utilities'
+import '../Info/InfoButton.css'
+
+const EuroFunds = ({data, toggledOn, toggle}) => {
+  return (
+    <div className="eurofunds info-button">
+      <Button outline color="primary" onClick={toggle}>
+        {toggledOn ? <ChevronUp aria-hidden="true" /> : <ChevronDown aria-hidden="true" />}
+        &nbsp;Eurofondy:&nbsp;<ShowNumberCurrency num={data.eufunds_price_sum} />&nbsp;
```

Comment:
Will this and `Contracts` differ in the near future? Maybe you could extract the repeated code to some wrapper component and pass just the text here as `children`, or something like that. What do you think?

Reply:
And also some function for extracting link and displayed text of course.

Reply:
Yeah, I noticed that (also `Notices` is similar, and even `Relations`), I'll think about it and create `InfoButton` component (there is already `InfoButton.scss` they share)